src/util/StringUtils.cpp

```diff
@@ -26,14 +26,14 @@ void StringSplit(const string &str,
   size_t cutAt;
   string tmp = str;
   while ((cutAt = tmp.find_first_of(delim)) != tmp.npos) {
-    if(cutAt > 0) {
+//    if(cutAt > 0) {
```

Comment:
Don't do this! This will break other modules. The right thing to do here is to change this function to have an extra argument "bool ignore_multiple_separators" and to do 

if (ignore_multiple_separators && cutAt > 0) ...

Then every time this function is called set ignore_multiple_separators=true (don't use default values) except in your use case (morph?) where ignore_multiple_separators=false.